.cargo/config.toml

```diff
@@ -0,0 +1,9 @@
+[alias]
+xtask = "run --package xtask --"
+make = "xtask make"
+asm = "xtask asm"
```

Comment:
Let's call this target `objdump` so it's clear what it does, WDYT?

Reply:
Cargo objdump is surely better than cargo asm, but this overlaps a command of widely used `cargo-binutils` tool at: https://github.com/rust-embedded/cargo-binutils/tree/master/src/bin .

Reply:
Oh interesting, I didn't know - good point :-)

Reply:
What is the difference between the objdumps? I can't see it yet.
Like, all toolchains have an objdump, be it GCC, LLVM, or the wrapper in Cargo.

Reply:
So this here just invokes `llvm-objdump` etc:
https://github.com/rust-embedded/cargo-binutils/blob/83cc6c14d40c6db463e10be3e36e1a43846d58cd/src/tool.rs#L43

Now `objdump`, for a given binary with metadata (e.g., ELF), it gives you the disassembled parts, whereas `asm`, on the other hand, may have people think of doing the forward direction, assembly.

Reply:
Ah, are you saying we cannot use `cargo objdump` as an alias because it already exists?

How do you feel about `disasm`?

Reply:
Let's take it as it is for now :)